src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -17,6 +17,8 @@ public static class ScriptConstants
         public const string AzureFunctionsColdStartKey = "MS_AzureFunctionsColdStart";
         public const string AzureFunctionsHttpRequestKey = "MS_AzureFunctionsHttpRequest";
         public const string AzureProxyFunctionExecutorKey = "MS_AzureProxyFunctionExecutor";
+        public const string AzureProxyFunctionLocalRedirectHeader = "MS_AzureProxyFunctionLocalRedirectHeader";
```

Comment:
This is the format we use for request properties, not headers. Recommend using "X-PROXY-REDIRECTS" or something. See the other "x-" headers defined in this file.